### PR TITLE
PR: Print chained exceptions

### PR DESCRIPTION
**Pull request by impact27**
Fixes https://github.com/spyder-ide/spyder/issues/2943
Fixes https://github.com/spyder-ide/spyder/issues/1052

### CODE REVIEW

File: spyder_kernels/console/kernel.py [hunk 1]
```diff
@@ -73,6 +73,53 @@ def __init__(self, *args, **kwargs):
...
 
+        # Hack to solve spyder-ide/spyder/2943
+        self.shell_showtraceback = self.shell.showtraceback
+        self.shell.showtraceback = self.chained_showtraceback
+
+    def chained_showtraceback(
+            self, exc_tuple=None, filename=None, tb_offset=None,
+            exception_only=False, running_compiled_code=False):
+        """
+        Display the exception that just occurred, and any chined exception.
```
Comment (goanpeca) [inline spyder_kernels/console/kernel.py]: chined -> chained

**Comment by ccordoba12** [comment]
@impact27, this is not the right way to do it. You simply need to remove this line

https://github.com/spyder-ide/spyder-kernels/blob/2c5b36cdb797b8aba77bc406ca96f5e079c4aaca/spyder_kernels/console/start.py#L103

But you also need to update the `TracebackLinksMixin` class in order for file strings in chained exceptions to be clickable and can be opened in our editor.

**Comment by impact27** [comment]
The result would be different:
Regular:
Screenshot 2019-11-29 at 06 10 26
New:
Screenshot 2019-11-29 at 06 12 22

**Comment by ccordoba12** [comment]
I'd really prefer that we switch to show tracebacks as it's done in a regular IPython session, to unify the behavior with the notebook and JupyterLab.

**Pull request closed without merging**